### ISSUE TITLE
Conduct Load adds empty config

### ISSUE THREAD

**Issue by fsat** [typesafehub/conductr-cli#521]
Somehow an empty config is created with simple `conduct load <bundle>`:

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ git checkout 1.2.16
HEAD is now at 4ea9739... Update version to 1.2.16

Felixs-MBP-2:conductr-cli felixsatyaputra$ sandbox run 2.1.2 --no-default-features
|------------------------------------------------|
| Stopping ConductR                              |
|------------------------------------------------|
ConductR core pid 62996 stopped
ConductR agent pid 63097 stopped
ConductR has been successfully stopped
|------------------------------------------------|
| Starting ConductR                              |
|------------------------------------------------|
Extracting ConductR core to /Users/felixsatyaputra/.conductr/images/core
Extracting ConductR agent to /Users/felixsatyaputra/.conductr/images/agent
Starting ConductR core instance on 192.168.10.1..
Waiting for ConductR to start..
Starting ConductR agent instance on 192.168.10.1..
|------------------------------------------------|
| OCI-in-Docker support unavailable.             |
|------------------------------------------------|
|------------------------------------------------|
| To provide support ensure Docker is running    |
| and restart the sandbox                        |
|------------------------------------------------|
|------------------------------------------------|
| Summary                                        |
|------------------------------------------------|
|- - - - - - - - - - - - - - - - - - - - - - - - |
| ConductR                                       |
|- - - - - - - - - - - - - - - - - - - - - - - - |
ConductR has been started:
  core instance on 192.168.10.1
  agent instance on 192.168.10.1
ConductR service locator has been started on:
  192.168.10.1:9008
|- - - - - - - - - - - - - - - - - - - - - - - - |
| Proxy                                          |
|- - - - - - - - - - - - - - - - - - - - - - - - |
HAProxy has not been started
To enable proxying ensure Docker is running and restart the sandbox
|- - - - - - - - - - - - - - - - - - - - - - - - |
| Bundles                                        |
|- - - - - - - - - - - - - - - - - - - - - - - - |
Check latest bundle status with:
  conduct info
Current bundle status:
Licensed To: cc64df31-ec6b-4e08-bb6b-3216721a56b@lightbend
Max ConductR agents: 10
ConductR Version(s): 0.1.0, 2.1.*
Grants: akka-sbr, cinnamon, conductr

ID  NAME  TAG  #REP  #STR  #RUN  ROLES

Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct load continuous-delivery
Retrieving bundle..
Resolving bundle using [bintray_resolver, docker_resolver]
Loading bundle from cache typesafe/bundle/continuous-delivery
Bintray credentials loaded from /Users/felixsatyaputra/.lightbend/commercial.credentials
Retrieving from cache /Users/felixsatyaputra/.conductr/cache/bundle/continuous-delivery-2.1.0-870ee7d6a4f5853229275cca14b604b8aca62a5e74819c0a092858cc75b6c186.zip
Loading bundle to ConductR..
[##################################################] 100%
Bundle 870ee7d6a4f5853229275cca14b604b8-d690a918e7fbc253faa2d06161be417e is installed
Bundle loaded.
Start bundle with:        conduct run 870ee7d-d690a91
Unload bundle with:       conduct unload 870ee7d-d690a91
Print ConductR info with: conduct info
Print bundle info with:   conduct info 870ee7d-d690a91
```

The the CD bundle is loaded with an empty config `d690a91`:

```
Felixs-MBP-2:tmp felixsatyaputra$ conduct info continuous-delivery
BUNDLE ATTRIBUTES
-----------------
Bundle Id              870ee7d-d690a91
Bundle Name            continuous-delivery
Compatibility Version  3
System                 continuous-delivery
System Version         3
Tags                   2.1.0
Nr of CPUs             0.1
Memory                 402653184
Disk Space             200000000
Roles                  continuous-delivery
Bundle Digest          870ee7d6a4f5853229275cca14b604b8aca62a5e74819c0a092858cc75b6c186
Configuration Digest   d690a918e7fbc253faa2d06161be417e446e81fe931b25c3fa5a529e8d402961
Error                  No

BUNDLE INSTALLATIONS
--------------------
Host                  192.168.10.1
Bundle                /Users/felixsatyaputra/.conductr/images/tmp/conductr/192.168.10.1/bundles/870ee7d6a4f5853229275cca14b604b8aca62a5e74819c0a092858cc75b6c186.zip
Bundle configuration  /Users/felixsatyaputra/.conductr/images/tmp/conductr/192.168.10.1/bundles/d690a918e7fbc253faa2d06161be417e446e81fe931b25c3fa5a529e8d402961.zip

Felixs-MBP-2:tmp felixsatyaputra$ cp /Users/felixsatyaputra/.conductr/images/tmp/conductr/192.168.10.1/bundles/d690a918e7fbc253faa2d06161be417e446e81fe931b25c3fa5a529e8d402961.zip .

Felixs-MBP-2:tmp felixsatyaputra$ unzip d690a918e7fbc253faa2d06161be417e446e81fe931b25c3fa5a529e8d402961.zip
Archive:  d690a918e7fbc253faa2d06161be417e446e81fe931b25c3fa5a529e8d402961.zip
 extracting: bundle/bundle.conf

Felixs-MBP-2:tmp felixsatyaputra$ cat bundle/bundle.conf
{}

```

**Comment by fsat** [comment]
Refer to #506 and #503 when starting the investigation.

**Comment by fsat** [comment]
Fixed by #524.
